package.json

```diff
@@ -27,7 +27,7 @@
   "license": "MIT",
   "dependencies": {
     "prop-types": "^15.6.2",
-    "tinymce": "^6.3.1"
+    "tinymce": "^6.0.0 || ^5.5.1"
```

Comment:
should this be "^6.3.1 || ^5.10.7" ?

or does that need a release version bump?

Reply:
No. It is intended to match the widest range of versions. We don't actually use TinyMCE, this is to provide the types that are shown when people type:
```tsx
<Editor init={{ some_tinymce_setting: 'blah' }} />
```
It will always default to the latest version of TinyMCE unless someone has specifically included their own version in a package.json in which case we use that.

Reply:
The only reason we have "^5.5.1" instead of "^5.0.0" is that was the first release with types.